packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/Converters.kt

```diff
@@ -292,7 +292,7 @@ internal inline fun realmObjectToRealmValueWithImport(
 
 // Returns a converter fixed to convert objects of the given type in the context of the given mediator/realm
 internal fun <T> converter(
-    clazz: KClass<*>,
+    clazz: KClass<T & Any>,
```

Comment:
Hmm, not sure I understand this change?

Reply:
The converter's `T` can be a nullable type e.g. `converter<String?>` but `KClass` has an upper bound of `Any` (which is not nullable), so this is a way of getting the always non-null part of `T`. With this change we are able to connect the `clazz` with other type references which wasn't possible before https://kotlinlang.org/docs/whatsnew17.html#stable-definitely-non-nullable-types